components/header/desktop-navigation.tsx

```
@@ -9,19 +9,28 @@ const MemoizedNavItem = memo(NavItem);
+              {title}
+            </MemoizedNavItem>
+          ))}
+        </ul>
+      </nav>
+    );
+  },
+  (prevProps, nextProps) => {
+    return (
+      prevProps.pages.length === nextProps.pages.length &&
+      prevProps.pages.every((page, index) => {
+        return (page.title = nextProps.pages[index].title);
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)